README.md

```diff
@@ -101,4 +102,10 @@ Type: `boolean`. Default: `false`.
 Enable if you need to add the old `:focus` instead of `:focus-visible` selector
 to every `:hover`.
 
+### `splitRules`
+
+Type: `boolean`. Default: `true`.
+
+Enable if you need to split the new selector into a separate rule. Use this if you need to support browsers, which doesn't support `:focus-visible`.
```

Comment:
1. You need to rewrite this sentence since it is now enabled by default.
2. And we need to change the example on top of the `README.md`
3. And in options example above we need `splitRules: false`